### PR TITLE
Replace 'Text' with 'str' in py3 stdlib

### PR DESCRIPTION
**Pull request by srittau**

### CODE REVIEW

File: stdlib/py_compile.pyi [hunk 1]
```diff
@@ -1,7 +1,5 @@
 import sys
-from typing import AnyStr, List, Optional, Text, Type, Union
-
-_EitherStr = Union[bytes, Text]
```
Comment (Akuli) [inline stdlib/py_compile.pyi:4]: Was this alias really never used?

Comment (srittau) [reply stdlib/py_compile.pyi:4]: I suspect it was used at some point, but not anymore.

**Review by Akuli — APPROVED** [review]
Let's wait until CI works again

**Comment by srittau** [comment]
@JelleZijlstra Thank you for merging. Unfortunately, GitHub Actions has been down since around 7AM UTC, so we should probably wait with further merges, since CI isn't running.

**Comment by JelleZijlstra** [comment]
Oops, sorry for missing that. I just saw green checkmarks and didn't look further.

**Comment by srittau** [comment]
Yeah, it's a bit of a misdesign that "CI is not running" is a "no op", instead of causing errors.

**Pull request merged**